boards/nxp/frdm_mcxn947/doc/index.rst

```diff
@@ -98,6 +98,8 @@ The FRDM-MCXN947 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | FLEXIO    | on-chip    | flexio                              |
 +-----------+------------+-------------------------------------+
+| SAI       | on-chip    | sai                                 |
```

Comment:
```suggestion
| I2S    | on-chip    | sai                                 |
```
Please change this interface name to I2S to match the Zephyr driver name.  Many Zephyr users will not be familiar with the SAI peripheral, or that it provides the I2S interface.

Reply:
actually the right column is what needs to match a zephyr driver name

Reply:
Thanks for the comments. I have modified it according to the index.rst of mimxrt1170_evk.